dev/fattest.simplicity/src/componenttest/rules/repeater/FeatureReplacementAction.java

```diff
@@ -347,6 +363,20 @@ private static String getReplacementFeature(String originalFeature, Set<String>
         return null;
     }
 
+    private static boolean removeWildcardFeature(Set<String> features, String removeFeature) {
+        String matcher = removeFeature.substring(0, removeFeature.length() - 1); //This assumes the feature string is in the format "feature-*"
```

Comment:
I don't think we should make this assumption that wildcards are always `feature-*`, mainly because if I do something slightly different but valid like `feature*` or `feature-1.*` it would be tricky to figure out why the features are not being removed.